modules/rules_gazebo/0.0.1/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+"rules_gazebo"
+
+module(
+    name = "rules_gazebo",
+    version = "0.0.1",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "buildifier_prebuilt", version = "6.1.2")
+bazel_dep(name = "rules_cc", version = "0.0.10")
```

Comment:
rules_cc isn't compatible with Bazel 6 in its latest version (I sent https://github.com/bazelbuild/rules_cc/pull/248 to change this). Could you try to downgrade the version? It's generally better to depend on lower versions if possible (for a library/ruleset) to allow consumers more flexibility.

Reply:
Downgraded to 0.0.9 in version 0.0.2

Also added non-module workflow test in upstream.